util/mkerr.pl

```diff
@@ -122,20 +122,22 @@ sub help
 }
 
 # Data parsed out of the config and state files.
+# We always map function-code values to zero, so items marked below with
+# an asterisk could eventually be removed.  TODO(4.0)
```

Comment:
So the script should determine whether the header is internal or not and then output the deprecated macro defines or not.